ui/selectmenu.js

```diff
@@ -498,6 +501,7 @@ return $.widget( "ui.selectmenu", {
 		// Change native select element
 		this.element[ 0 ].selectedIndex = item.index;
 		this.buttonItem.replaceWith( this.buttonItem = this._renderButtonItem( item ) );
+		this._addClass( this.buttonItem, "ui-selectmenu-text" );
```

Comment:
This is duplicated in `_drawButton`. Could be avoided by moving it inside `_renderButtonItem`.